hedera-node/hedera-app-spi/src/main/java/com/hedera/node/app/spi/api/ServiceApiProvider.java

```diff
@@ -24,11 +25,13 @@ public interface ServiceApiProvider<T> {
      *
      * @param configuration  the node configuration
      * @param writableStates the writable state of the service
-     * @param entityCounters
+     * @param entityCounters the entity counters
+     * @param nodeFeeAccumulator the accumulator for node fees (used for in-memory fee accumulation)
      * @return the new API instance
      */
     T newInstance(
             @NonNull Configuration configuration,
             @NonNull WritableStates writableStates,
-            @NonNull final WritableEntityCounters entityCounters);
+            @NonNull WritableEntityCounters entityCounters,
+            @NonNull NodeFeeAccumulator nodeFeeAccumulator);
```

Comment:
* <i>(FUTURE)</i> Once HIP-1259, "Fee Collection Account", is enabled by default, we could replace 
     * {@code TokenServiceApiImpl}'s injected {@code NodeFeeAccumulator} by instead passing the
     * {@code NodeFeeManager} to the existing {@code ObjLongConsumer<AccountID>} argument. (The current
     * use of that argument to track <b>all</b> accumulated node fees for HIP-1065 could then be replaced
     * by just summing the fees from all account ids in the {@code NodePayment} singleton.)

Reply:
Thank you. Done Michael!